### ISSUE TITLE
Clarify the status when there are no available peers

### ISSUE THREAD

**Issue by daviddoria** [ProjectSPAN/android-manet-ptt#2]
When I launch the app, I am greeted with a picture of a microphone and a dropdown box with options "Silence Mode" and "Listen Only Mode". It was not clear that when a peer was available it would appear directly in this list. After looking through the code, I see that the list is populated with these two defaults, along with any peers and any groups.

Perhaps this "Select a Mode, Group, or Peer" dialog could have three distinct sections - "Passive" (for silence and listen only), "Peers", and "Groups". When no one else is connected, "Peers" and "Groups" should somehow indicate "[none available]".

**Comment by jrobble** [comment]
I agree, that from a usability standpoint it would be nice to divide the drop down into sections and show when there's no peers and no groups; however, at the time I wrote the code the Android Spinner widget wasn't that sophisticated. Ultimately, we should probably use a Button that launches a ListView instead. ListViews are easier to work with, especially when dealing with dynamically updating content.
